test/cluster/mv/test_mv_building.py

```
@@ -196,6 +221,86 @@ async def progress_saved():
+        # Restart with new shard count
+        if interrupt_resharding:
+            # Start, wait for resharding to begin, then kill and restart.
+            # Prevent view building worker from resuming work on view building.
+            await manager.server_update_config(server.server_id, "error_injections_at_startup", ["dont_start_build_step"])
+            log = await manager.server_open_log(server.server_id)
+            mark = await log.mark()
+            await manager.server_start(server.server_id, cmdline_options_override=cmdline_after)
+            try:
+                await log.wait_for("Reshard", from_mark=mark, timeout=30)
+            except:
+                pass  # resharding might not always appear in logs for small data
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

(SIM105)